roles/dlrn_report/tasks/main.yml

```diff
@@ -14,11 +14,21 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Warning if DLRN is not running
+  debug:
+  msg: "DLRN will not be executed if it is a testproject, make sure you set cifmw_dlrn_report_force_report to true if you want to report to DLRN"
+  when:
+    - ('testproject' in zuul.project.name) and (cifmw_dlrn_report_force_report | bool)
```

Comment:
If I understand correctly, this condition is wrong.
You want it to display for testproject, but when the force variable is set to false, right?
Then the value of force variable should be `not`ed I think?

Right now in testproject without force variable (i.e. using default value) the whole task will be skipped.
`[(testproject – true) and (force false)] => false`